portfolio/src/main/java/com/google/sps/servlets/FileUploadServlet.java

```diff
@@ -32,22 +40,60 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList; 
 
 /** Servlet responsible for processing request with URL from Blobstore. */
 @WebServlet("/upload")
 public class FileUploadServlet extends HttpServlet {
 
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("File").addSort("timestamp", SortDirection.DESCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
```

Comment:
what exactly is results?

Reply:
The results are all the files in the server because I queried for "File" entities